plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/model/SQLServerTriggerBase.java

```diff
@@ -49,8 +50,17 @@
     private String type;
     private String body;
     private long objectId;
+    private String typeDescription;
+    private String triggerTypeDescription;
+    private String eventGroup;
+    private Date createDate;
+    private Date modifyDate;
     private boolean insteadOfTrigger;
     private volatile int disabled;
+    private boolean isMsShipped;
+    private boolean isNotForReplication;
+    private boolean isFirst;
+    private boolean isLast;
```

Comment:
Since these fields are assigned in constructor and they can't be changed later, mark them `final`.